pandas/io/tests/test_pickle.py

```diff
@@ -76,21 +87,18 @@ def read_pickles(self, version):
                 if 'ts' in data['series']:
                     self._validate_timeseries(data['series']['ts'], self.data['series']['ts'])
                     self._validate_frequency(data['series']['ts'])
-
-    def test_read_pickles_0_10_1(self):
-        self.read_pickles('0.10.1')
-
-    def test_read_pickles_0_11_0(self):
-        self.read_pickles('0.11.0')
-
-    def test_read_pickles_0_12_0(self):
-        self.read_pickles('0.12.0')
-
-    def test_read_pickles_0_13_0(self):
-        self.read_pickles('0.13.0')
-
-    def test_read_pickles_0_14_0(self):
-        self.read_pickles('0.14.0')
+            n += 1
+        assert n > 0, 'Pickle files are not tested'
+
```

Comment:
rather than making these in one test, can you have a routine which create tests from the legacy_pickle dir (like you are doing now), but makes them separate tests (so need to do this at import time). That way you can see them running indepedently (and if they fail they would fail independetly)